src/TTrigger.cpp

```diff
@@ -750,6 +750,24 @@ bool TTrigger::match_lua_code(int regexNumber)
     return false;
 }
 
+bool TTrigger::match_prompt(int patternNumber)
+{
+    if (mpHost->mpConsole->mIsPromptLine) {
+        if (mudlet::debugMode) {
+            TDebug(QColor(Qt::yellow), QColor(Qt::black)) << "Trigger name=" << mName << "(" << mRegexCodeList.value(patternNumber) << ") matched.\n" >> 0;
```

Comment:
Consider redoing in a `QStringLiteral(`...`)` or `tr(`...`)` form - we'll only have to hunt lines like this down and do that later... oh and does `patternNumber` need a `+1` or `-1` to match the 1-50 range in the Editor GUI?

🤔 BTW does `TDebug()` ever use the `>> 0;` bit or is that some esoteric requirement of providing for the use of `<<` and `>>` operators in that they have to be utilised even when not useful?

Reply:
Hm, I'll leave the reformatting of the message for when we do start fixing all the other instances.

I have no idea about `>> 0;`.

Reply:
🤔 That sounds like something for another PR - as I said I do not _think_ we ever use whatever the `>> 0;` is about and it just clutters up the current places where the `TDebug()` {display things on the _Central Debug Console_} class is used.